README.md

```diff
@@ -7,9 +7,7 @@ This is a sample project that can be used to jump start your Syna project. It us
 To start using this starter you need to download this repository and update it's git submodules (Syna theme is added as a submodule).
 
 ```
-git clone git@github.com:okkur/syna-start.git project-name && cd $_
-git submodule init
-git submodule update
+git clone --recurse-submodules git@github.com:okkur/syna-start.git project-name
```

Comment:
Maybe the https url is easier as it won't ask for any key details/password. Also we assume people won't push back any changes anyway.